app/assets/stylesheets/admin.scss

```
@@ -0,0 +1,103 @@
+@charset "utf-8";
+
+@import "normalize-rails";
+@import "variables";
+@import "flashes";
+
+header {
+  display: flex;
+  background: $hotline-bling-pink;
+  padding: 0 1em;
+  margin-bottom: 2em;
+}
+
+main {
+  font-size: 18px;
+  margin: 0 auto;
+  max-width: 80%;
+}
+
+.redirection {
+  border-bottom: 1px solid black;
+  display: flex;
+  flex-direction: column;
+  justify-items: center;
+  margin-bottom: 1em;
+}
+
+.chunk {
```

Not a big deal, but would it be nicer to avoid all the `chunk` class declarations and redefine this as:

```suggestion
.redirection > div {
```